### DISCUSSION TITLE
How to 'separate' settings in the database from content?

### DISCUSSION THREAD

**Discussion by udippel** [gerbera/gerbera#3175] [General]
With reference to #3123, I'd like to ask for advice:

Played with some things, and managed to crash gerbera. My problem. 
At restart, it finds plenty of database problems, and shuts down. Still no general problem. 
Somehow I managed to enter the web-interface, though. Alas, it shows the available files, but the database view comes with a red status line, so there seems to be no way to actually access the entry "PC directory". Therefore, no way to delete its content.

And here my question comes in: How can one save one's (database) settings, when for database problems, the "PC directory" isn't visible? 

Any idea will be appreciated. "Starting from scratch" is frustrating, when one has settings in the database ... .
... Any SQL command to forcibly remove the 'root' of that "PC directory" in the database, so that the content will be reconstructed after a restart of gerbera, eventually?

**Comment by KarlStraussberger** [comment]
You can find the database schema in https://docs.gerbera.io/en/stable/dev-database.html.

The easiest way is to backup the contents of `grb_config_value` and delete the whole database. Restart to have the database recreated. Afterwards you can reload your config values one by one to avoid breaking gerbera again.

**Comment by udippel** [comment]
Thanks a lot! Will try that ... .
